community/ndp/messaging-v1/src/main/java/org/neo4j/ndp/messaging/v1/PackStreamMessageFormatV1.java

```diff
@@ -225,10 +225,13 @@ public void flush() throws IOException
         private void packRawMap( Map<String,Object> map ) throws IOException
         {
             packer.packMapHeader( map.size() );
-            for ( Map.Entry<String,Object> entry : map.entrySet() )
+            if(map.size() > 0)
```

Comment:
I could only understand this `if` is added for compiler-performance (branch-prediction in binary code level) purpose. But what I actually want to see is the formatting is wrong here.